src/resource_clients/run.ts

```diff
@@ -138,6 +141,35 @@ export class RunClient extends ResourceClient {
         return cast(parseDateFields(pluckData(response.data)));
     }
 
+    /**
+     * TODO: docs url
+     * https://github.com/apify/apify-client-js/issues/614
```

Comment:
can we add the URL here right ahead? even if it doesn't exist yet, feels better than publishing with a TODO

Reply:
I don't have the URL. I'll ask Mat'o what's the status on the endpoint docs 👍

Reply:
I am rather sure we can guess it before it exists, its just another endpoint, the URLs for them are not random.